### DISCUSSION TITLE
Chrome first-time run blocks tests

### DISCUSSION THREAD

**Discussion by cambiph** [leancodepl/patrol#1818] [Q&A]
When you run tests on a fresh device and perform an action that opens a webview, the Chrome first-time setup is shown:
<img width="427" alt="image" src="https://github.com/leancodepl/patrol/assets/1628243/0351323d-5e94-4a04-a2a2-54973d3535e0">
This is a blocking action because the WebView stays open and we are unable to go back to the app.

Is there a way to circumvent this?

**Comment by bartekpacia** [comment]
You have to tap on `Accept & continue` button, there's no way around it.
